Monitoring user workloads feature. (#25)
openshift/managed-openshift-docs: assemblies/understanding-the-monitoring-stack.adoc

[id="understanding-the-monitoring-stack"]
= Understanding the monitoring stack
include::_attributes/attributes-openshift-dedicated.adoc[]
:context: understanding-the-monitoring-stack

[role="_abstract"]
User-workload monitoring lets you monitor your own applications using a different Prometheus instance on your own worker nodes, without interfering with Red Hat Site Reliability Engineer (SRE) platform metrics.

[NOTE]

> Follow the instructions in this section carefully to configure a supported Prometheus instance. Custom Prometheus instances are not supported by the SREs.

Reviewer: ```suggestion
Follow the instructions in this section carefully to configure a supported Prometheus instance. Custom Prometheus instances are not supported on OpenShift Dedicated.
```

Author: Changed it to {product-title} to accomodate OSD and ROSA.
